### PR TITLE
Add metric template for new PRs

### PR DESCRIPTION
**Pull request by AakashKumarNain**

### CODE REVIEW

File: tensorflow_addons/metrics/README.md [hunk 1]
```diff
@@ -10,6 +10,86 @@ must:
  * Inherit from `tf.keras.metrics.Metric`.
  * Register as a keras global object so it can be serialized properly: `@tf.keras.utils.register_keras_serializable(package='Addons')`
 
+#### Sample Template
+
+Any PR submitted for adding a new metric to the TF-addons ecosystem must ensure that it covers all the major points listed here.
```
Comment (seanpmorgan) [inline tensorflow_addons/metrics/README.md:15]:
```suggestion
#### Metric Requirements

Any PR which adds a new metric must ensure that the following three cases are well tested and supported:
```

**Comment by AakashKumarNain** [comment]
Thanks @seanpmorgan for the review. I have updated the changes suggested
